activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/ReplayServiceTaskRequest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.rest.api;
+
+import javax.validation.constraints.NotEmpty;
+
+public class ReplayServiceTaskRequest {
```

Comment:
I think this class should be moved to api module. We can also remove it in favor of using path variables to simplify , i.e. `POST /admin/v1/service-task/{executionId}/{flowNodeId}/replay`

Reply:
As long we don't include a specific method in the Java API it should be fine to keep it at the same level as the controller. Am I missing something?